index.js

```diff
@@ -0,0 +1,264 @@
+  <ul class="skils">
+    <li class="button">HTML}</li>
+    <li class="skil button">bootstrap</li>
+    <li class="skil button">Ruby</li>
+  </ul>
+  </div>
+  <button class ="see-project-2" id="seeproject3">see my project</button>
+  `;
+cardContainer.innerHTML += projectM4;
+const projectM5 = `<div class="card-2">
+  <h2 class="prof-art">Profesional Art Printing Data</h2>
+  <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard</p>
+  <ul class="skils">
+    <li class="button">HTML}</li>
+    <li class="skil button">bootstrap</li>
+    <li class="skil button">Ruby</li>
+  </ul>
+  </div>
+  <button class ="see-project-2" id="seeproject4">see my project</button>
+  `;
+cardContainer.innerHTML += projectM5;
+const projectM6 = `<div class="card-2">
+  <h2 class="prof-art">Profesional Art Printing Data</h2>
+  <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard</p>
+  <ul class="skils">
+    <li class="button">HTML}</li>
+    <li class="skil button">bootstrap</li>
+    <li class="skil button">Ruby</li>
+  </ul>
+  </div>
+  <button class ="see-project-2" id="seeproject5">see my project</button>
+  `;
+cardContainer.innerHTML += projectM6;
+
+// Get the popup mobile
+const modal = document.getElementById('myModal');
+modal.innerHTML = `<div class="modal-content">
+<div class="modalHead-close">
+<h2 class = "modalHeader" id="modalHeader">Multi Post Stories</h2>
+ <span class="close">&times;</span>
+ </div>
+ <div>
+  <ul id="modalTech" class ="modalTech">
+      <li>HTML</li>
+      <li>Bootstrap</li>
+      <li>Ruby on rails</li>
+  </ul>
+  <img src="/Portfolio.png" alt="popup image">
+  <p class = "modal-pm" >Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent</p>
+  <div class = "live-source">
+  <a href="https://www.google.com/">see live <img src="/Icon.png" alt="google"></a>
+  <a href="https://github.com/">see source <img src="/Vector.png" alt=""></a>
+  </div>
+</div>
+</div>`;
+// Get the button that opens the modal
+const btn0 = document.getElementById('seeproject0');
+const btn1 = document.getElementById('seeproject1');
+const btn2 = document.getElementById('seeproject2');
+const btn3 = document.getElementById('seeproject3');
+const btn4 = document.getElementById('seeproject4');
+const btn5 = document.getElementById('seeproject5');
+const btn6 = document.getElementById('seeprojectt');
+
+// Get the <span> element that closes the modal
+const span = document.getElementsByClassName('close')[0];
```

Comment:
It is a very good and recommended practice to name your variables with a meaningful names that describes the content it holds otherwise it will be misleading and makes your code hard to understand